packages/js/product-editor-extension/templates/src/blocks/example-block/block.json.mustache

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Noting that we differ from `@wordpress/create-block` here which spins up a new block.  The purpose of these templates is not to spin up a generic block, but to show off specific use cases so specific block metadata and naming is provided.